websites/A/AnimeDigitalNetwork/presence.ts

```diff
@@ -11,17 +11,15 @@ presence.on("UpdateData", async () => {
     presenceData: PresenceData = {
       largeImageKey: "logo"
     };
-
+  
   if (document.location.pathname.includes("video") && video) {
-    if (video && !isNaN(video.duration)) {
-      const title = document.querySelector(
-          "#root > div > div > div.sc-pbWVv.hTvDIL > div > div:nth-child(1) > div.sc-kbKFCX.sc-kgMcbC.kecmmo > div:nth-child(1) > div > div > h1 > a"
-        ),
-        timestamps = presence.getTimestamps(
+    const episode = JSON.parse(document.querySelector("#root > div > div > div.sc-pkSvE.kPCOPp > div > div:nth-child(1) > script").textContent);
+    if (!isNaN(video.duration)) {
+      const timestamps = presence.getTimestamps(
           Math.floor(video.currentTime),
           Math.floor(video.duration)
         );
```

Comment:
might as well use presence.getMediaTimestamps or it's called something like that.